[Writing by hand. 22:40 of 27.01.2021.]
Created this folder (just a few seconds ago).
Different writings of "Ⰺ" are possible.

[28.01.2021: 15:12]
Creating a file "1.txt" instead of "_1.txt" (it was "1.txt"; at "data/").
Changing there some things:
 * "," and ";"
 * raws.
 * Some words. Ex.: "я был, была " -> "я был; я была "
 * Adding comments. Ex.: "рыже-бурый (о лисице) " -> "рыже-бурый [(о лисице) ]".
 ... * Adding some lines there.

[partially 29-30.01.2021]
Starting writing the worker with translatioon and transliteration to cyr. and glag.
in main, using the data of previous writings and codes.

[20:33 of 30.01.2021]
Starting writing from Sublime Text 3. (Before it was here just a little.)

[01.02.2021]
[~16:57]
Replacing `polling` with `infinity_polling`. Test.

[17:00]
Launc.~

... {Maybe some wasn't written here.}

[08.02.2021 1:53-55+]
Adding the `-` dash to pattern of ~WORD at game `words`.
#!! Is it written correctly?

[2:18]
Adding `cmd_pattern` at `/words skip`, wasn't.

[2:33+]
Registering at PythonAnyWhere site.

[2021-02-12: ~0:19]
Changing to @TransToOldSlavonic_bot.

[12.02.2021]
* Placing bot @TransToOldSlavonic_bot to the chat (and chats), removing
previous bot partially.

[13.02.2021]
Starting the correcting process. E. g.: transliteration of numbers to cyryllic,
exceptions while playing words.

[15.02.2021 0:33]
Writing the function that transliterates a number to cyryllic. Possible error/
mind to: SOME NUMBERS MAY BE QUITE DIFF. (i. e., for example, "i" at Eng. and
Ukr. are chars-table-different).

[18.02.2021 1:18]
~? At another dir (сopied, edited)
Added Procfile. Trying to collected all required for launch and test at Heroku.
* Set config vars at Heroku just a few minutes ago.

! [2:07]
! Deployed to Heroku. Test. /help worked.

Turned the PC off -> fail.

Trying to set `web`->`worker` at Procfile.
[Fail.]

Putting `web`.

[2:37]
Created new app, deleted previous. This was launched with the answer to message
in the Telegram.

More then 2 minutes lasted. Turning the PC off.

[~2:50 (>2:49, <2:50)]
`heroku run:detached python worker.py -a translation-test-2` -> :

2021-02-18T00:49:24.500383+00:00 app[api]: Starting process with command `python worker.py` by user me2001.com@gmail.com
2021-02-18T00:49:28.267495+00:00 heroku[run.1678]: Starting process with command `python worker.py`
2021-02-18T00:49:28.862179+00:00 heroku[run.1678]: State changed from starting to up
2021-02-18T00:49:30.636661+00:00 app[run.1678]: Parsing the start of configs` file.

...
TMP
---
LICENSE, README, .gitignore
Procfile (web: ...) ?
runtime.txt

[18.02.2021T16:12]
Placing `tokens.py` to the `data/`.

[19.02.2021]
Restarted at 11:15. Prev. version worked for ~24 hours, crashed by time soon
after the try to launch 2 get-updates at one time.

~[11:35-11:38]
Was inactive... Isn't correct?

[02/27/21T17:15:16] detected the bot as failed, relaunched it just now.

[2021-03-07T00:55]
Changed:
at `runtime.txt`: `3.8.7`->`3.8.8` (was warned via CLI)
